lib/gooddata/models/project.rb

```diff
@@ -483,12 +483,12 @@ def transfer_schedules(from_project, to_project)
       end
 
       def transfer_tagged_stuff(from_project, to_project, tag)
-        puts "Transferring tagged stuff - #{tag}"
+        GoodData.logger.info("Transferring tagged stuff - #{tag}")
 
         objects = from_project.find_by_tag(tag)
 
         if objects.any?
-          puts JSON.pretty_generate(objects)
+          GoodData.logger.info(JSON.pretty_generate(objects))
```

Comment:
wouldn't it be better to start with a newline?

Reply:
OK, newline added.